_includes/master/reqs-sys.md

```diff
@@ -31,19 +31,24 @@ to connect to, we provide instructions in the [installation documentation](./ins
 
 ## Network requirements
 
-{{site.prodname}} requires the network to allow the following types of traffic.
-
-| Traffic | Protocol | Port |
-| ------- | -------- | ---- |
-| BGP     | TCP      | 179  |
-| IPIP\*  | 4        | n/a  |
-
-\* {% if include.orch == "OpenStack" %} If your compute hosts connect directly
-   and don't use IPIP, you don't need to allow IPIP traffic. {% else %} Our
-   manifests enable IPIP by default.  If you disable IPIP, you won't need to
-   allow IPIP traffic. {% endif %} Refer to [Configuring
-   IP-in-IP](../../usage/configuration/ip-in-ip) for more information.
-
-> **Tip**: On GCE, you can allow this traffic using firewall rules. In AWS, use
-> EC2 security group rules.
-{: .alert .alert-success}
+Ensure that your hosts and firewalls allow the following traffic.
+
+| Configuration                                                                         | Host                | Connection type | Port/protocol                                                 |
+|---------------------------------------------------------------------------------------|---------------------|-----------------|---------------------------------------------------------------|
+| {{site.prodname}} networking                                                          | All                 | Bidirectional   | TCP 179                                                       |
+| {{site.prodname}} networking in IP-in-IP mode (default mode)                          | All                 | Bidirectional   | IP-in-IP, often represented by its protocol number `4`        |
+{%- if include.orch == "Kubernetes" %}
+| flannel networking (VXLAN)                                                            | All                 | Bidirectional   | UDP 4789                                                      |
+| etcd datastore                                                                        | etcd hosts          | Incoming        | Often TCP 2379 but varies per etcd configuration              |
+| {{site.prodname}} networking, Kubernetes API datastore, cluster of more than 50 nodes | Typha agent hosts   | Incoming        | TCP 5473 (default)                                            |
+{%- else %}
+| All                                                                                   | etcd hosts          | Incoming        | Varies per etcd configuration                                 |
+{%- endif %}
+{%- if include.orch == "Kubernetes" or include.orch == "OpenShift" %}
+| All                                                                                   | kube-apiserver host | Incoming        | Often TCP 443, but check the `port` value returned by `kubectl get svc kubernetes -o yaml` |
```

Comment:
Sorry, got this one slightly wrong - it's 6443 by default. See here: https://kubernetes.io/docs/admin/accessing-the-api/#api-server-ports-and-ips

Reply:
>  the `port` value returned by `kubectl get svc kubernetes -o yaml`

This is also a strange way to go about it, I think. We probably just want to tell them to check the apiserver configuration?

e.g. "The value passed to the apiserver `--secure-port` option"

Reply:
Yes, I have been looking at this page quite a bit. That's just the thing, I think there are probably three different common possibilities: 443, 6443, and 8443 (openshift). I can mention all three , it will help them some, but not a lot. Also, I think rather than making them try to remember what might have been passed into the api server, it will be easier to give them this command which helps them in real time.